src/components/CategoryPage.js

```diff
@@ -9,9 +9,9 @@ const CategoryPage = ({ activities, title, overviewText, importanceText, keySkil
             container
             direction="column"
             justifyContent="center"
-            alignItems="center"
+            alignItems="left"
```

Comment:
This change also affects the activity blocks, which doesn't look good left-aligned. Undo this change and add a Grid with alignItems="left" around just the Typography as paragraphs.
<img width="1680" alt="Screen Shot 2022-04-20 at 1 00 29 AM" src="https://user-images.githubusercontent.com/43453743/164154057-3b7694d7-7ed7-43a1-b564-e45decd33f08.png">